src/composables/useRaycaster/index.ts

```diff
@@ -16,7 +16,16 @@ export const useRaycaster = (
   const canvas = computed(() => ctx.renderer.value.domElement as HTMLCanvasElement)
   const intersects: ShallowRef<Intersection[]> = shallowRef([])
   const { x, y } = usePointer({ target: canvas })
-  const objectWihtEvents = computed(() => objects.value.filter(obj => obj.__tres?.eventCount > 0))
+  const objectWihtEvents = computed(() => {
+    const hasEvents = (object)=> object.__tres?.eventCount > 0;
+    const checkObject = (object)=> {
+      if (object.isGroup) {
+        return object.children.some(checkObject)
+      }
+      return hasEvents(object)
+    }
+    return objects.value.filter(checkObject)
+  })
```

Comment:
Hey @hexianWeb, thank you for the issue and the PR with a fix!

The solution looks good as is, but I'd like to clean it up a bit for readability

I know it's subjective on whether ternaries *are* more readable, so feel free to push back if you don't like this suggestion

```suggestion
  const hasEvents = (object) => object.__tres?.eventCount > 0;
  const isGroup = (object) => object.isGroup ? object.children.some(isGroup) : hasEvents(object)

  // Recurse through scene graph and filter for objects with events
  const objectWihtEvents = computed(() => objects.value.filter(isGroup))
```

Also, while this fixes the `Groups` case, any `Object3D` can contain children as well, so maybe the fix will look more like this?

```suggestion
  const hasEvents = (object) => object.__tres?.eventCount > 0;
  const hasChildren = (object) => object?.children.length > 0 ? object.children.some(hasChildren) : hasEvents(object)

  // Recurse through scene graph and filter for objects with events
  const objectWihtEvents = computed(() => objects.value.filter(hasChildren))
```

And 2 things that you didn't cause but would be nice if you could fix

1. Can we fix the `objectWihtEvents` typo? `objectWithEvents`
2. We're filtering for the events in the raycaster, but the source of these objects is coming from the EventManager. Can we move this logic into it's computed instead? [Link](https://github.com/Tresjs/tres/blob/main/src/composables/useTresEventManager/index.ts#L33)

Reply:
Sorry for replying so late, I have changed the code. 
Please let me know if any code needs to be modified.